team/index.html

```diff
@@ -572,6 +581,7 @@ <h2 class="avenirDemi">ncs@jssaten.ac.in</h2>
 	</section>
 	</div>
 	<!-- 	New page 2019 -->
+      <script src="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.13.0/js/all.min.js"></script>
```

Comment:
Don't use the whole library for just 5 icons. You should use SVGs. Search [this](https://icons8.com/icons/) website you will find all the svgs here.

Reply:
ok sir